substrate/utils/frame/remote-externalities/src/lib.rs

```diff
@@ -240,20 +240,20 @@ pub struct OnlineConfig<B: BlockT> {
 	pub hashed_keys: Vec<Vec<u8>>,
 }
 
-impl<B: BlockT> OnlineConfig<B> {
+impl<H: Clone> OnlineConfig<H> {
 	/// Return rpc (http) client reference.
 	fn rpc_client(&self) -> &HttpClient {
 		self.transport
 			.as_client()
 			.expect("http client must have been initialized by now; qed.")
 	}
 
-	fn at_expected(&self) -> B::Hash {
-		self.at.expect("block at must be initialized; qed")
+	fn at_expected(&self) -> H {
+		self.clone().at.expect("block at must be initialized; qed")
```

Comment:
@ggwpez not sure about this change

Reply:
Yes it is more correct than previous - which pulled in `Copy`. I think its good 👍